tests/behat/features/capabilities/administration/view-filtered-users.feature

```diff
@@ -0,0 +1,73 @@
+@account @profile @stability @stability-4 @perfect @api @3245778 @view-filtered-users
```

Comment:
@navneet0693 Nice work with this test 🚀 It helped to catch this issue https://www.drupal.org/project/social/issues/3202509